examples/multiplayer-lan/godot/README.md

```diff
@@ -0,0 +1,17 @@
+# Multiplayer Lan Example
+
+This is a sample multiplayer example showing off how to set up a lobby + dedicated server with godot-rust.
+
+
+
+Largely adapted from FinePointCGI's Godot multiplayer tutorial:
```

Comment:
why are we adapting youtube tutorial instead of one of godot's demo projects? https://github.com/godotengine/godot-demo-projects/tree/master/networking